src/pages/tasks/TaskAssigneeSelectorModal.tsx

```diff
@@ -220,14 +217,14 @@ function TaskAssigneeSelectorModal({reports, task}: TaskAssigneeSelectorModalPro
                     />
                     <View style={[styles.flex1, styles.w100, styles.pRelative]}>
                         <SelectionList
-                            sections={didScreenTransitionEnd && !isLoading ? sections : []}
+                            sections={(!areOptionsInitialized && didScreenTransitionEnd) || areOptionsInitialized ? sections : []}
```

Comment:
I think we can do it similar to `NewChatPage` as I suggested above

```suggestion
                            sections={areOptionsInitialized ? sections : []}
```
and update `useOptionsList`:

```javascript
                             const {options, areOptionsInitialized} = useOptionsList({
        shouldInitialize: didScreenTransitionEnd,
    });
```